app/controllers/devise/sessions_controller.rb

```diff
@@ -77,7 +77,11 @@ def respond_to_on_destroy
     # support returning empty response on GET request
     respond_to do |format|
       format.all { head :no_content }
-      format.any(*navigational_formats) { redirect_to after_sign_out_path_for(resource_name) }
+      format.any(*navigational_formats) { redirect_to_after_sign_out_path }
     end
   end
+
+  def redirect_to_after_sign_out_path
+    redirect_to after_sign_out_path_for(resource_name), allow_other_host: true
+  end
```

Comment:
I am not about what's the best place to write tests for this. If someone can point me in a direction I'll be happy to add tests 😃.

Reply:
Well, after thinking about this a bit longer, I wonder if this proposed solution is actually a good one 😅. 

Different use cases might require different options to be passed to the `redirect_to`. It's probably not possible to cover all such cases by hard coding options here. So, the question that comes to my mind now is:  **What's the best way to override the options passed to the `redirect_to`?**  For example, to also change the redirection status code etc. 

• Override the `destroy` action of the controller. Then, having full control over the redirection (already possible).
• Provide a `after_sign_out_redirect_options_for` method or similar where users can define the custom options they want for the redirection?
• Other?

Reply:
Or set the default in Devise in the meantime to `allow_other_host: true` and put a warning in a comment in after_sign_up_path_for about overriding it with a user provided URLs? You have to go out of your way to override the method, so it doesn't seem like a less secure default.